### DISCUSSION TITLE
Top 50 scores should be highlighted in recent feed as it was on the old site

### DISCUSSION THREAD

**Discussion by Rdzeniu** [ppy/osu#18167] [Ideas] [answered]
I can't be the only one that liked when my top 50 score was standing out on my profile.
Of course it is not necessary and probably most of you guys don't care but I think it's cool :P 

![before](https://user-images.githubusercontent.com/1345628/167345006-f1a191df-c72c-4e62-b8c8-9b0cb7aee20c.png)
![opera_WnZp8kY3xT](https://user-images.githubusercontent.com/1345628/167345036-22b0499f-de66-4dba-a7ed-2b3771bc9a7e.png)

**Comment by smoogipoo** [comment, marked as answer]
This is something you should request at https://github.com/ppy/osu-web instead.
